network/load-balancer/reference-content/configuring-load-balancer.mdx

```diff
@@ -0,0 +1,137 @@
+
+## Configuring a Load Balancer for SSL bridging
+
+SSL bridging allows the user to initiate a secure HTTPS connection with the Load Balancer thanks to the Load Balancer frontend's SSL certificate. The Load Balancer uses its SSL certificate to decrypt incoming HTTPS traffic. This allows the Load Balancer to carry out layer 7 actions on the received traffic. The Load Balancer's backend then initiates a new HTTPS connection to re-encrypt traffic between the Load Balancer and the backend server, this time using the backend server's certificate. 
+
+    <Lightbox src="scaleway-ssl-bridging.webp" alt="" />
+
+To configure your Load Balancer for SSL bridging:
+
+- The frontend should listen on port 443.
+- The frontend must have a [certificate](/network/load-balancer/how-to/add-certificate).
+- The backend should be configured to initiate connections using port 443 and HTTPS protocol.
+- The backend server should have its own [certificate](/network/load-balancer/concepts#certificate).
+
+## Configuring a Load Balancer for SSL offloading
+
+SSL offloading, also known as SSL termination, allows the user to initiate a secure HTTPS connection with the Load Balancer thanks to the Load Balancer frontend's SSL certificate. The Load Balancer uses its SSL certificate to decrypt incoming HTTPS traffic. Layer 7 actions may therefore be applied to the traffic at this stage. Traffic is not re-encrypted on its way from the Load Balancer to the backend server, unlike with SSL bridging. Traffic that has gone through the offloading process is marked with a new header, called `X-Forwarded-Proto`.
+
+    <Lightbox src="scaleway-ssl-offloading.webp" alt="" />
+
+To configure your Load Balancer for SSL offloading:
+
+- The frontend should listen on port 443.
+- The frontend must have a [certificate](/network/load-balancer/how-to/add-certificate).
+- The backend should be configured to initiate connections using port 80 and HTTP protocol.
+- The backend server does not need its own certificate.
+
+## Configuring a Load Balancer for SSL passthrough
+
+Passthrough is the simplest way to handle HTTPS traffic on a Load Balancer. As the name suggests, traffic is simply passed through the Load Balancer without being decrypted on it. Whilst this option generates very low overhead, no layer 7 actions can be carried out. This means that no cookie-based sticky sessions are possible with this method. In addition, if an application does not share sessions between servers, users’ sessions may get lost by being redirected to different servers of the group. 
+
+    <Lightbox src="scaleway-ssl-passthrough.webp" alt="" />
+
+To configure your Load Balancer for SSL passthrough:
```

Comment:
I am not 100% sure about the configuration here but to me:

- Frontend can listen on whatever port you want
- Backend Must be configured with TCP protocol, and the port can be whichever you want, the only constraint is that the backend server listens on this port with its HTTPS server process, and of course has a certificate

Reply:
done